code/src/components/GameScreen.js

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import styled from 'styled-components';
+
+import { generateAction } from '../reducers/games';
+
+const GameWrapper = styled.div`
+  margin-top: 100px;
+  background-color: darkgreen;
+  padding: 20px;
+  font-size: 12px;
+  width: 50vw;
+  word-wrap: break-word;
+  
+  @media (min-width: 768px) {
+    font-size: 15px;
+  }
+
+  @media (min-width: 1200px) {
+    font-size: 20px;
+  }
+`
+const ActionsDiv = styled.div`
+  border-bottom: 5px dotted white;
+  color: lightgreen;
+`
+const DescriptionDiv = styled.div`
+  border-bottom: 10px dotted white;
+  color: white;
+`
+const DirectionButton = styled.button`
+  border: none;
+  background-color: lightgreen;
+  border-radius: 50px;
+  margin-bottom: 20px;
+`
+const EndText = styled.p`
+  color: white;
+`
+
+const GameScreen = () => {
+    const dispatch = useDispatch();
+    let [direction, setDirection] = useState('');
+
+    const directionArray = [
```

Comment:
Interesting with this local stage! I notice that you also have this directionArray in reducers line 35 - 40. Wondering if that could just have been exported instead.